test/models/API1811/DocumentDetailsSpec.scala

```diff
@@ -22,6 +22,33 @@ import utils.API1811.TestConstants._
 
 class DocumentDetailsSpec extends SpecBase {
 
+  "getAccruingPenalty" should {
+
+    "return only the penalty charge that has an accruing penaltyStatus" in {
+
+      val model = fullDocumentDetails.copy(
+        documentPenaltyTotals = Some(Seq(documentPenaltyTotalsPosted, documentPenaltyTotals)))
+
+      model.getAccruingPenalty shouldBe Some(documentPenaltyTotals)
+    }
+
+    "return an empty DocumentPenaltyTotals when PenaltyStatus is not accruing" in {
```

Comment:
Would you mind checking this - in your test a `None` is returned rather than an empty `DocumentPenaltyTotals` - seems like it may be just the description is wrong?